src/applications/disability-benefits/2346/sass/form-2346.scss

```diff
@@ -28,6 +28,9 @@
       font-size: 16px;
     }
 }
+.confirmation-page > .usa-alert-success::before {
+    color: #000;
```

Comment:
Please fix this, it is a sass linting warning

![image](https://user-images.githubusercontent.com/2008881/83583440-36b70f00-a4f9-11ea-967d-219910fdc243.png)

Reply:
@bkjohnson Okay I have pushed an update.